bazel/foreign_cc/dlb.patch

```diff
@@ -1,34 +1,3 @@
-diff --git a/dlb/libdlb/Makefile b/dlb/libdlb/Makefile
-index e7fa1fa..a9f566e 100644
---- a/dlb/libdlb/Makefile
-+++ b/dlb/libdlb/Makefile
-@@ -9,7 +9,7 @@ BUILD_DIR := build
- CFLAGS  := -g -fPIC -DVERSION_$(VERSION) -Wall -O3 -pthread
- CFLAGS  += -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security \
-            -Werror=format-security -fstack-protector-strong
--CFLAGS  += -fno-delete-null-pointer-checks -fno-strict-overflow -fwrapv
-+CFLAGS  += -fno-delete-null-pointer-checks -fwrapv
- CFLAGS  += $(EXTRA_CFLAGS)
-
- ifeq ($(DLB_DISABLE_DOMAIN_SERVER), TRUE)
-@@ -33,7 +33,7 @@ SRCS :=         \
-
- OBJS := $(addprefix $(BUILD_DIR)/, $(SRCS:.c=.o))
- DEPS := $(addprefix $(BUILD_DIR)/, $(SRCS:.c=.d))
--LIB  := libdlb.so
-+LIB  := libdlb.a
```

Comment:
Please confirm this didn't introduce a dynamic dependency? The 7.8 build defaults to static linkage?

Reply:
Yeah, it is. Now still build static linkage for DLB.